### PR TITLE
add use-path grammar

### PR DESCRIPTION
**Pull request by cardoso**
This seems to be missing from #174 . I just took a rough look at wit-parser and this seems to be what it's doing currently. 

cc @Mossaka @alexcrichton

### CODE REVIEW

**Review by Mossaka — APPROVED** [review]
Thanks! Lgtm

**Review by alexcrichton — COMMENTED** [review]
Thanks! Could `use-path` reuse the preexisting `interface` production perhaps? That may require renaming `interface`, however, since `include` refers to worlds instead of interfaces.

**Comment by cardoso** [comment]
@alexcrichton can it take the version as well? The parser accepts it, but I didn't see ui tests for it. If the parser is right, I can improve this and add more tests there.

**Comment by cardoso** [comment]
@alexcrichton can it take the version as well? The parser accepts it, but I didn't see ui tests for it. If the parser is right, I can improve this and add more tests there.

**Comment by cardoso** [comment]
Okay, so it seems they're the same:

```wit
interface ::= id
            | id ':' id '/' id ('@' valid-semver)?
```

 If this is it, then I'll open the PR for the parser once this is merged.

**Comment by cardoso** [comment]
It does seem confusing though since it should refer to worlds, but I'll leave it to you.

File: design/mvp/WIT.md [hunk 1]
```diff
@@ -927,8 +927,8 @@ include my-world-2
 ```
 
 ```ebnf
-include-item ::= 'include' use-path
-               | 'include' use-path 'with' '{' include-names-list '}'
+include-item ::= 'include' interface
+               | 'include' interface 'with' '{' include-names-list '}'
```
Comment (cardoso) [inline design/mvp/WIT.md:931]: Would it make more sense to rename interface to use-path instead?

Comment (alexcrichton) [reply design/mvp/WIT.md:931]: I think so yeah, but either way works too

**Comment by cardoso** [comment]
Thanks @alexcrichton . I think it reads better now (and uses the same naming as wit-parser).

**Review by alexcrichton — APPROVED** [review]

**Comment by lukewagner** [comment]
Thanks!

**Pull request merged**